content/1.7/back-office/international/locations/zones-listing.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Could you move this file in the "zones" folder: content/1.7/back-office/international/locations/zones/zones-listing .md ?